discord/commands.py

```diff
@@ -42,45 +39,51 @@ def __new__(cls, *args, **kwargs):
 
         self.__original_kwargs__ = kwargs.copy()
         return self
+
     def __init__(self, func, *args, **kwargs):
         if not asyncio.iscoroutinefunction(func):
-            raise TypeError('Callback must be a coroutine.')
+            raise TypeError("Callback must be a coroutine.")
+        self.callback = func
 
-        name = kwargs.get('name') or func.__name__
+        self.guild_ids = kwargs.get("guild_ids", None)
```

Comment:
Should it be defaulted to list?

Reply:
It should be defaulted to `None` to make the command global